experimental/LieAlgebras/src/CartanMatrix.jl

```diff
@@ -61,52 +77,52 @@ function cartan_matrix(fam::Symbol, rk::Int)
       mat = mat[1:7, 1:7]
     end
   elseif fam == :F
-    @req rk == 4 "type Fn requires rank rk to be 4"
+    @req rk == 4 "type Fn requires rank to be 4"
     mat = matrix(ZZ, [2 -1 0 0; -1 2 -1 0; 0 -2 2 -1; 0 0 -1 2])
   elseif fam == :G
-    @req rk == 2 "type Gn requires rank rk to be 2"
+    @req rk == 2 "type Gn requires rank to be 2"
     mat = matrix(ZZ, [2 -3; -1 2])
   else
-    error("unknown family $fam")
+    throw(ArgumentError("unknown family"))
   end
 
   return mat
 end
 
-function cartan_matrix(types::Tuple{Symbol,Int}...)
-  blocks = ZZMatrix[cartan_matrix(type...) for type in types]
-
-  return block_diagonal_matrix(blocks)
-end
+@doc raw"""
+    cartan_matrix(type::Tuple{Symbol,Int}...) -> ZZMatrix
 
-function cartan_to_coxeter_matrix(mat; check=true)
-  @req !check || !is_cartan_matrix(mat) "$mat is not a (generalized) Cartan matrix"
+Returns a block diagonal matrix of indecomposable Cartan matrices as defined by type. For allowed values see `cartan_matrix(fam::Symbol, rk::Int)`.
```

Comment:
```suggestion
Returns a block diagonal matrix of indecomposable Cartan matrices as defined by type.
For allowed values see `cartan_matrix(fam::Symbol, rk::Int)`.
```
break overly long line